jsk_naoqi_robot/jsk_pepper_startup/apps/meeting/scripts/meeting.l

```diff
@@ -0,0 +1,194 @@
+   "golden-saying-by-pepper-3.png"
+   "golden-saying-by-pepper-4.png"
+   "golden-saying-by-pepper-5.png"
+   "golden-saying-by-pepper-6.png"))
+(defvar *pose-list* (list *pose1* *pose2* *pose3* *pose4* *pose5* *pose6*))
+(defvar *time-list* (list *time1* *time2* *time3* *time4* *time5* *time6*))
+(defvar *comment-list*
+  (list
+   "\\vol=40\\\\vct=120\\\\rspd=80\\研究会に出て，\\vct=150\\えらいッ！\\rspd=100\\"
+   "\\vct=120\\\\rspd=80\\ご飯，食べた？\\rspd=100\\"
+   "\\vct=120\\\\rspd=80\\今週も，\\vct=150\\頑張ろうッ！\\rspd=100\\"
+   "\\vct=120\\\\rspd=80\\コメント少ないのは，順調ってことだよッ！\\rspd=100\\"
+   "\\vct=120\\\\rspd=80\\座りっぱなし，\\vct=150\\気をつけてッ！\\rspd=100\\"
+   "\\vct=120\\\\rspd=80\\報告して，\\vct=150\\えらいッ！\\rspd=100\\"
+   ))
+
+;; parameter
+(defparameter *continue* t)
+(defparameter *count* 0)
+(defparameter *last-tm* (ros::time-now))
+
+(ros::roseus "online-meeting-enhancement-main")
+
+(ros::subscribe "joy" sensor_msgs::Joy
+                #'(lambda (msg)
+                    (let ((button (send msg :buttons)))
+                      (ros::ros-info "received joy button ~A" button)
+                      (when (eq (elt button 3) 1)
+                        (send *ri* :speak "はい")
+                        (setq *continue* nil)))))
+
+(defun start-func (args)
+  (let ()
+    (setq *last-tm* (ros::time-now))
+    ;;
+    (send *ri* :servo-on)
+    (send *ri* :set-language "Japanese")
+    (send *ri* :stop-grasp)
+    (send *ri* :set-basic-awareness-enabled nil)
+    (send *ri* :set-background-movement-enabled nil)
+    (send *ri* :show-image (elt *img-list* 0))
+    (send *ri* :speak (elt *comment-list* 0))
+    (send *ri* :angle-vector-sequence (elt *pose-list* 0) (elt *time-list* 0))
+    (send *ri* :wait-interpolation)
+    (send *ri* :set-basic-awareness-enabled t)
+    (send *ri* :set-background-movement-enabled t)
+    ;;
+    (setq *count* 1)
+    (set-alist 'description "Start Meeting" args)
+    :started))
+
+(defun end-func (args)
+  (let ()
+    (send *ri* :show-image "golden-saying-by-pepper-7.png")
+    (send *ri* :angle-vector-sequence (list #f(2.0 -2.0 -5.0 85.0 20.0 -70.0 -20.0 -40.0 85.0 -20.0 70.0 20.0 40.0 30.0 -10.0) #f(2.0 -2.0 -5.0 85.0 10.0 -70.0 -20.0 -40.0 85.0 -10.0 70.0 20.0 40.0 0.0 0.0) #f(2.0 -2.0 -5.0 85.0 20.0 -70.0 -20.0 -40.0 85.0 -20.0 70.0 20.0 40.0 -30.0 -10.0) #f(2.0 -2.0 -5.0 85.0 10.0 -70.0 -20.0 -40.0 85.0 -10.0 70.0 20.0 40.0 0.0 0.0)) (list 1000 1000 1000 1000))
+    (send *ri* :speak "\\vct=120\\\\rspd=80\\みなさん")
+    (send *ri* :wait-interpolation)
+
+    (send *ri* :angle-vector-sequence (list *a* *d* *a* *d* *a* *d* *a* *d* *a* *e* *reset*) (list 1000 1000 1000 1000 1000 1000 1000 1000 1000 1000 3000))
+    (send *ri* :speak "\\vct=120\\\\rspd=80\\おつかれさまでしたッ！\\rspd=100\\")
+    (send *ri* :wait-interpolation)
+    (send *ri* :hide-image)
+    (send *ri* :set-language "English")
+    ;;
+    (set-alist 'description "End Meeting" args)
+    :finished))
+
+(defun comment-func (args)
+  (let (count str (wait 600) tm)
```

Comment:
In `jsk_pepper_startup/apps/meeting/scripts/meeting.l`,
I found the wait time was 60 sec, but right time should be 600 sec.
If the wait time is 60 sec, Pepper speaks too many times.

I modified this, but I mistakenly edited it directly without sending a pull-request... Sorry for that.

Reply:
@kochigami thank you!